Nighttime-Operations/Weather-Constraints/index.rst

```diff
@@ -0,0 +1,179 @@
+The weather server on Gemini updates with new information every 30 seconds, so wind speeds are gusts, rather than sustained averages.
+
+Go outside and look for incoming cloud fronts, and inspect the buildings and cars for condensation if humidity is above 70%.
+Be more cautious to open if there will not be someone in the control room to montior changing weather conditions, e.g. opening before going to dinner or to complete another task.
+
+.. warning::
+    Vent gates:
+        The vent gates on the first floor of the AuxTel dome are not rated for high winds. 
+        The vent gates MUST be closed if the wind speed (gusts and sustained) is above 8 m/s.
+
+
+
+.. note::
+    Extraction fan use:
+        The extraction fan on vent gate 3 can operate at 50-100% power for initial cooling. 
+        During on-sky operations, the extraction fan must be run at 20-25% power.
+        If winds are over 8 m/s, the vent gate must be closed and the extraction fan turned off.
+        If humidity is rising quickly, or >= 65%, the extraction fan should be turned off to avoid drawing in more moisture through the dome. 
+
+.. _weather-constraints-wind:
+
+Wind
+^^^^
+The predominant wind direction at the summit is from the northwest or ~300 degrees.  
+The majority of the time, the wind comes up from the coast and blows through the Elqui River valley, arriving to the summit. 
+If the direction changes frequently, or has completely changed direction from 300 degrees, use caution as this likely indicates a frontal system and unstable atmospheric conditions.
+
+Wind speeds between 0 and 12 m/s (gusts):
+    - This is safe operation range for AuxTel. 
+        No restrictions in observing are needed. 
+        
+Wind speeds between 12 and 15 m/s (gusts):
+    - If the observer has control to choose targets, observe out of the wind.
+        Report that the scheduler configuration needs to be changed, or the observing program should be changed. 
+        Avoid targets that are pointing direction into or 180 degrees opposite of the dominant direction of the wind.
+
+.. note::
+    90 degree rule:
+        Due to the curved dome, pointing at targets 180 degrees offset from the dominant wind direction creates a wind eddy under the lip of the dome, which can create wind shake on the upper truss of AuxTel. 
+        To compensate for this, it is recommended to look for targets that have a 90-degree offset from the dominant wind direction. 
+
+Wind speeds greater than 15 m/s (gusts):
+    - Observers should consider closing once wind gusts have become greater than 15 m/s. 
+        Sustained winds (gusts averaged over 5 minutes) of more than 15 m/s should initiate telescope closure. 
+        Before deciding to open again, ensure that sustained winds have dropped below the closure limits for more than 15 minutes. 
+
```

Comment:
I think we got paranoid that one time around 10 m/s. It juts out quite a bit, so I would tend to be more conservative, esp. since it just lets in more wind to shake the truss. Added. 👍